packages/theme/src/components/ui/input/HelperText.vue

```diff
@@ -1,7 +1,30 @@
 <template>
   <div
-    class="text-sm text-neutral-700"
+    class="text-sm text-right"
+    :class="{'text-neutral-700': remaining >= 0,
+      'text-red-600': remaining < 0
+    }"
   >
-    <slot />
+  <slot>
+   {{ remaining }}
+  </slot>
   </div>
 </template>
+
+<script setup lang="ts">
+const props = defineProps({
+  value: {
+    type: String,
+    default: "",
+    required: true,
+  },
+  maxLength: {
+    type: Number,
+    default: 0,
+    required: true,
+  },
```

Comment:
The `HelperText.vue` component is generic component we might use for various use-cases.
Passing a specific `maxLength` props does no make sense.

For starter, you can simply pass `isError` props and handle the logic directly in PostEditor.